src/guide/getting_started.md

```diff
@@ -7,6 +7,9 @@ Integrating Breez SDK - *Liquid* into your application takes just a few minutes.
 - **[Listening to events](/guide/events.md)**
 - **[Adding logging](/guide/logging.md)**
 
+## API Key
+Some methods require the Breez API key to be set in order to work. You can request one for free by filling our form <a target="_blank" href="{{api_key_form_uri}}">here</a>.
```

Comment:
Clicking this link in the generated mdbook gives a 404, as it tries to access `/guide/{{api_key_form_uri}}`.